src/main/java/org/jboss/logmanager/LogContext.java

```diff
@@ -505,6 +509,7 @@ public void close() throws Exception {
                     }
                 }
             }
+            pinnedSet.clear();
```

Comment:
This breaks if `Set.of()` is used with an `UnsupportedOperationException`.

Reply:
Ah, easy fix. One moment.